python/ray/autoscaler/load_metrics.py

```diff
@@ -51,8 +51,8 @@ def update(self,
                 static_resources != dynamic_resources:
             self.last_used_time_by_ip[ip] = now
         self.last_heartbeat_time_by_ip[ip] = now
-        self.waiting_bundles = waiting_bundles
-        self.infeasible_bundles = infeasible_bundles
+        self.waiting_bundles_by_ip[ip] = waiting_bundles
+        self.infeasible_bundles_by_ip[ip] = infeasible_bundles
```

Comment:
Why is it by IP? Isn't it a global thing for the entire cluster?

Reply:
ermmm i was under the impression that each raylet was reporting its own infeasible tasks. can @stephanie-wang confirm?